bundles/org.openhab.core.io.rest.publisher/src/main/java/org/openhab/core/io/rest/publisher/internal/JAXRSConnectorProvider.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.io.rest.publisher.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thirdparty.com.eclipsesource.jaxrs.publisher.internal.JAXRSConnector;
+
+/**
+ * A provider of an JAX RS connector.
+ *
+ * @author Markus Rathgeb - Initial contribution
+ */
+@NonNullByDefault
+public interface JAXRSConnectorProvider {
```

Comment:
You have chosen `OhJaxRsResource` for the other interface, but `JAXRSConnectorProvider` here. Can we reach an agreement about the casing?

Reply:
You are absolutely right. We need to be consistent.
We are using most of the case upper case if the official abbreviation does the same (e.g. DTO and IP), there are also one exception you pointed below (Sse) but I think upper case maies sense.

Reply:
Changes `OhJaxRsResource` to `OhJAXRSResource`.